test/runtests.jl

```diff
@@ -369,11 +369,11 @@ end
 @testset "Broadcasting" begin
     temp = deepcopy(ca)
     @test eltype(Float32.(ca)) == Float32
-    @test ca .* ca' == cmat
+    @test ca * ca' == cmat
     @test 1 .* (ca .+ ca) == ComponentArray(a .+ a, getaxes(ca))
```

Comment:
We don’t want to get rid of tests for broadcasted extrusion. All of these should stay as dot multiplication. We already have tests for linear algebra. These are equivalent mathematically, but not programmatically.